src/odo.ts

```diff
@@ -298,7 +298,7 @@ export class OdoImpl implements Odo {
         const result: cliInstance.CliExitData = await this.execute(
             Command.printOdoVersionAndProjects(), process.cwd(), false
         );
-        if (result.stdout.indexOf('Please log in to the cluster') > -1 || result.stdout.indexOf('the server has asked for the client to provide credentials') > -1) {
+        if (result.stdout.indexOf('Please login to your server') > -1 || result.stdout.indexOf('the server has asked for the client to provide credentials') > -1) {
```

Comment:
The odo(0.0.16) output shows the following message during login: `Please log in to the cluster`.

Reply:
not for me, it doesn't

Reply:
right, there are in fact two spots in odo code that fail on login with different messages

Please log in:
https://github.com/redhat-developer/odo/blob/eec9a26ec4b7520e90a1d94370c3ce92f84d5f59/pkg/occlient/occlient.go#L246

Please login:
https://github.com/redhat-developer/odo/blob/eec9a26ec4b7520e90a1d94370c3ce92f84d5f59/pkg/occlient/occlient.go#L182 

No idea what the differentiating factor is.

Reply:
Hah!
```

Reply:
We should open an issue on the odo side, WDYT ?

Reply:
We need to open an issue on odo side then.

Reply:
I don't think this is odo's fault, it makes sense to have different errors for these two cases. We just haven't realized this path exists in odo.

Reply:
It might be just message from openshift and it could be different for different versions.
But anyway it worth to open issue for odo.